app/src/main/res/values/strings.xml

```diff
@@ -86,4 +86,5 @@
     <string name="toast_login_failed_fetching_backends">Failed to fetch Backends!</string>
     <string name="login_activity_toast_login_error">Unknown login error.</string>
     <string name="no_topic_in_message">(no topic)</string>
+    <string name="toast_text">Message Copied</string>
```

Comment:
Ideally the name of the string would more closely match it's contents.  For example, a more explicit name for this string would be "message_copied".